packages/ai/src/prompt/wrap-gateway-error.ts

```diff
@@ -1,21 +1,31 @@
-import {
-  GatewayAuthenticationError,
-  GatewayModelNotFoundError,
-} from '@ai-sdk/gateway';
+import { GatewayAuthenticationError } from '@ai-sdk/gateway';
 import { AISDKError } from '@ai-sdk/provider';
 
 export function wrapGatewayError(error: unknown): unknown {
-  if (
-    GatewayAuthenticationError.isInstance(error) ||
-    GatewayModelNotFoundError.isInstance(error)
-  ) {
+  if (!GatewayAuthenticationError.isInstance(error)) return error;
+
+  const isProductionEnv = process?.env.NODE_ENV === 'production';
+  const moreInfoURL = 'https://v6.ai-sdk.dev/unauthenticated-ai-gateway';
```

Comment:
will change to `https://ai-sdk.dev/unauthenticated-ai-gateway'` in backport to v5